worker-macros/src/durable_object.rs

```diff
@@ -32,6 +41,8 @@ pub fn expand_macro(tokens: TokenStream) -> syn::Result<TokenStream> {
                     "new" => {
                         let mut method = impl_method.clone();
                         method.sig.ident = Ident::new("_new", method.sig.ident.span());
+                        method.vis = Visibility::Inherited;
```

Comment:
I spent two hours trying to debug a problem caused by adding `pub` to any of the methods in the impl. It causes them to get also picked up by the `#[wasm_bindgen]` deceleration which generates extremely obtuse errors about WSAM ABI traits not being satisfied. Given that the wrapped method is public and is exported to JS I think it makes sense to allow, if not enforce that these methods are `pub`.

Reply:
If not forcing the visibility to `Inherited` we should at least generate a helpful error messages that the visibility cannot be `pub`.